optimum/habana/utils.py

```diff
@@ -400,3 +400,9 @@ def get_device_name():
         return "gaudi2"
     else:
         raise ValueError(f"Unsupported device: the device type is {device_type}.")
+
+
+def is_gaudi3():
+    import habana_frameworks.torch as htorch
+
+    return htorch.hpu.get_device_name() == "GAUDI3"
```

Comment:
@pbielak why do you make a new function? Can you use the get_device_name() directly? No need to add more code